ui/src/context/catalog.ts

```diff
@@ -1,11 +1,12 @@
 import axios from 'axios';
 import { UC_API_PREFIX } from '../utils/constants';
 
-const client = axios.create({
+/**
+ * The singleton client instance with `baseURL`.
+ */
+export const CLIENT = axios.create({
```

Comment:
I'm not sure renaming this file from `client` to `catalog` makes sense. What is your line of thinking?